packit_service/worker/events/new_hotness.py

```diff
@@ -215,13 +215,17 @@ def __init__(
         self._versions = versions
 
     @property
-    def version(self) -> str:
-        # TODO: Handle here or further down the chain? we should be able to get
-        # the package config from dist-git and resolve the next version; how are
-        # we going to choose the version?
-        #   a) latest greatest
-        #   b) next unreleased?
-        # this can be also influenced by the mask…
-        #
-        # It would be ideal to handle here because of the serialization…
-        raise NotImplementedError()
+    def version(self) -> Optional[str]:
+        return None
```

Comment:
I am not sure I understand why we return `None` here, can we add a comment? 🙏

Reply:
sure, will add

Reply:
added